packages/server/tests/acceptance/hbarLimiter.spec.ts

```diff
@@ -887,5 +914,77 @@ describe('@hbarlimiter HBAR Limiter Acceptance Tests', function () {
         });
       });
     });
+
+    describe('@hbarlimiter-batch3 BASIC Tier', () => {
```

Comment:
I had to add it to another branch because of the order of loading environment variables in the acceptance tests.  The `relay.ts` reads and sets the `HBAR_RATE_LIMIT_TINYBAR` at boot time.

Reply:
I believe he is addressing the scenario in which the limiter is turned off, allowing an account to make calls without being HBAR rate limited.

@ebadiere, if that's true and the suite is needed, I suggest renaming the test suite from "BASIC Tier" to something more descriptive, such as "Unlimited Limit" or "Limiter Off" or something like that. The term "BASIC Tier" is confusing, as we have another suite with the same name.

Reply:
@quiet-node Done.